crowdfunding_compassion/models/crowdfunding_participant.py

```diff
@@ -39,6 +39,7 @@ class CrowdfundingParticipant(models.Model):
     instagram_url = fields.Char(string="Instagram link")
     personal_web_page_url = fields.Char(string="Personal web page")
     profile_photo = fields.Binary(related="partner_id.image")
```

Comment:
With the new implementation, this field is only used to compute the value of the variable `profile_photo_url`. Both fields could be merged to avoid storing duplicated information.

Reply:
A related field is only a shortcut and does not store the information twice, so that's not really an issue here, unless it would have the attribute `store=True`

Reply:
Oops, I missed this `related=...` option, my bad !